docs/clients.md

```diff
@@ -108,4 +109,23 @@ if err != nil {
 
 ## Java
 
-TODO
+It is not recommended and supported to "assemble" a client in code,
+please use config file. The default config path is `audit_logging.yml`, or
+if it is overwritten via env var, AUDIT_CLIENT_CONFIG_PATH.
+
+### Create a client from a config file
+
+```java
+Injector injector = Guice.createInjector(new AuditLoggingModule());
+
+// For gRPC services, use AuditLoggingServerInterceptor.
+AuditLoggingServerInterceptor interceptor =
+        injector.getInstance(AuditLoggingServerInterceptor.class);
+
+// Otherwise, use LoggingClient directly.
+LoggingClient client = injector.getInstance(LoggingClient.class);
+```
+
+### Extend
```

Comment:
Let's have the same subsections as Go. But under "### "Assemble" a client in code" we say what you say at the top: "It is not supported..."